app/assets/javascripts/browsered/web-payments/index.js

```diff
@@ -1,36 +1,46 @@
 'use strict'
 
-const { prepareRequestObject, clearErrorSummary } = require('./helpers')
+const { clearErrorSummary } = require('./helpers')
 const makeApplePayRequest = require('./apple-pay')
-const makePaymentRequest = require('./google-pay')
+const { createGooglePaymentRequest, googlePayNow } = require('./google-pay')
 
 module.exports = () => {
   const paymentMethodForm = document.getElementById('payment-request-container')
   const standardMethodContainer = document.getElementById('enter-card-details-container')
 
   if (window.PaymentRequest && paymentMethodForm) {
```

Comment:
what does `paymentMethodForm` do?

Reply:
its an element on the page, it’s only there if the service has made webpayments available (a boolean stored in the DB) so this is how the clientside JS works out whether to trigger for webpayments, it’s stored as a `const` because it’s quicker to use the variable than look for the element everytime we want to do something with it.